src/NexusMods.App/Program.cs

```diff
@@ -43,6 +43,9 @@ public static int Main(string[] args)
             loggingSettings
         );
 
+        // Okay to do wait here, as we are in the main process thread.
+        host.StartAsync().Wait();
```

Comment:
Shouldn't main just be async instead? Or are there reasons we can't do that?

Reply:
I see, it appears to be running fine on my end, but I found the `[STAThread]` problems online, that it doesn't get applied correctly to main function that is generated out of the async one.